views/render-footer-tracking-events.php

```diff
@@ -0,0 +1,7 @@
+<?php if ($this->has_events_in_cookie): ?>
+    <script type="text/javascript">
+        jQuery(document).ready(function($) {
```

Comment:
I recall a discussion to ditch `jQuery` as a dependancy for our plugin, no?

Reply:
Yes but at the moment i cannot seem to be able to clear the cookie data without this script.

Reply:
I don't mean that you should remove the `onload` effect of this code, but rather that there is a pure javascript way of doing the same thing. Then you'll remove a whole library from loading.